src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -47,6 +50,11 @@ def load_arguments(self, _):
         c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
         c.argument('disable_proxy', options_list=['--disable-proxy'], arg_group='Proxy', action='store_true', help='Disables proxy settings for agents')
         c.argument('auto_upgrade', options_list=['--auto-upgrade'], help='Flag to enable/disable auto upgrade of arc agents. By default, auto upgrade of agents is enabled.', arg_type=get_enum_type(["true", "false"]))
+        c.argument('container_registry_repository', options_list=['--pcr-repository'], help='Custom container registry repository.')
+        c.argument('container_registry_username', options_list=['--pcr-username'], help='Custom container registry repository username.')
+        c.argument('container_registry_password', options_list=['--pcr-password'], help='Custom container registry repository password.')
+        c.argument('default_registry', options_list=['--default-registry'], action='store_true', help='Flag to restore default container registry (MCR).')
```

Comment:
let's not introduce another parameter for this. if the customer doesn't pass --container-registry-repository in the connect or update commands, we should just take that as an indicator that they are not going for their own container registry

Reply:
same reason as above